### PR TITLE
explorer: adding support to display arch decorator on numbers/offsets

### PR DESCRIPTION
**Pull request by mike-hunhoff**
Adding support to display new architecture decorator for `number` and `offset` features:

snippy_1

### CODE REVIEW

File: capa/ida/explorer/model.py [hunk 1]
```diff
@@ -537,16 +578,30 @@ def render_capa_doc_feature(self, parent, feature, location, doc, display="-"):
...
+        if feature["type"] in (
+            "bytes",
+            "api",
+            "mnemonic",
+            "number",
+            "offset",
+            "number/x32",
+            "number/x64",
+            "offset/x32",
+            "offset/x64",
```
Comment (mike-hunhoff) [inline capa/ida/explorer/model.py:603]: The only real change for this PR.

File: capa/ida/explorer/model.py [hunk 2]
```diff
@@ -353,13 +372,15 @@ def render_capa_doc_statement_node(self, parent, statement, locations, doc):
             # TODO: do we display 'not'
             pass
         elif statement["type"] == "some":
-            return CapaExplorerDefaultItem(parent, statement["count"] + " or more")
+            return CapaExplorerDefaultItem(parent, str(statement["count"]) + " or more")
```
Comment (mike-hunhoff) [inline capa/ida/explorer/model.py:375]: Unrelated bug fix.

**Comment by mr-tz** [comment]
looks like you had a field day with black 😄

**Comment by mike-hunhoff** [comment]
> looks like you had a field day with black 😄

Fixed with a `rebase` 😆
